src/main/java/org/craftercms/studio/impl/v2/upgrade/operations/AbstractContentTypeUpgradeOperation.java

```diff
@@ -148,6 +148,9 @@ protected boolean shouldBeUpdated(final String site, final Path file) throws Upg
             } else {
                 return true;
             }
+        } catch (ExecutionException e) {
```

Comment:
Shouldn't we just continue with any exception? Why are we stopping the whole pipeline if we can't parse an XML?

Reply:
I don't think it would be ok to skip an operation if it fails, the site would be left in an unknown state but it would look like it was updated, something that fails could cause even more errors in future upgrades or in the site itself

Reply:
We should log errors and continue

Reply:
@sumerjabri should this be only for sites or for the db & config too?

Reply:
Log errors and continue, except for DB errors or global/system-wide errors.